data/en/isdefined.json

```diff
@@ -3,22 +3,28 @@
 	"type":"function",
 	"syntax":"isDefined(variable_name)",
 	"returns":"boolean",
-	"related":["structKeyExists","isEmpty"],
-	"description":" Evaluates a string value to determine whether the variable\n named in it exists.",
+	"related":["structKeyExists","isEmpty","ArrayIsDefined"],
+	"description":"Evaluates a string value to determine whether the variable named in it exists.\n NOTE: Passing an Array entry {isDefined('thisArray[2]')} will error. Use ArrayIsDefined() instead",
 	"params": [
-		{"name":"variable_name","description":"","required":true,"default":"","type":"string","values":[]}
+		{"name":"variable_name","description":"The quoted string of the variable to test for existence.","required":true,"default":"","type":"string","values":[]}
 	],
 	"engines": {
 		"coldfusion": {"minimum_version":"3", "notes":"", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-in-k/isdefined.html"},
 		"lucee": {"minimum_version":"", "notes":"", "docs":"http://docs.lucee.org/reference/functions/isdefined.html"},
 		"railo": {"minimum_version":"", "notes":"", "docs":"http://railodocs.org/index.cfm/function/isdefined"},
 		"openbd": {"minimum_version":"", "notes":"", "docs":"http://openbd.org/manual/?/function/isdefined"}
 	},
-	"links": [],
+	"links": [
+		{
+		    "title":"structKeyExists() vs isDefined()",
+		    "description":"An Adobe Forums discussion on the differences between structKeyExists() and isDefined().",
+		    "url":"https://forums.adobe.com/thread/491991"
+		}
+	],
 	"examples": [
 		{
 			"title":"Using IsDefined",
-			"description":"Checking for the existence of a form variable",
+			"description":"Checking for the existence of a variable named 'form.submit' or a variable named 'submit' in the 'form' object.",
```

Comment:
They sound the same but they aren't quite. `isDefined()` does a lot of scope hunting, so it doesn't necessarily look for `variable` in the `form` scope. If it finds 'form.variable' in the `variables` scope, it will return true even if it doesn't exist in `form`. But I haven't played with `isDefined()` in a very long time. I don't know if the behavior has changed. When I get home later, I'll try to add a repro for that behavior.

Reply:
Also just noticed that this is for my `parameterExists()` PR. This file wasn't supposed to be included in this one. They were separate. I apparently borked my commit on this one. I need to remove the file from this PR.

Reply:
Maybe you should make that clear with a comparison example or the like.

Reply:
Done.